tests/common/platform/check_all_interface_info.py

```diff
@@ -0,0 +1,20 @@
+"""
+Helper script for checking all related information of interfaces
+
+This script contains re-usable functions for checking status of interfaces on SONiC.
+"""
+import logging
+from check_transceiver_status import all_transceivers_detected
+from check_interface_status import check_interface_status
```

Comment:
Just thinking out loud here, but since we're re-organizing these shared fixtures anyways maybe it would be a good time to think about how we're naming/organizing the modules?

For example, I feel like it might make more sense to call `check_transceiver_status` something like `transceiver_status` or `transceiver_utils` to indicate that it is a collection of functions for checking the transceiver rather than a script that checks the transceiver status.

Thoughts?

Reply:
This could also help us avoid the problem of creating giant flat directories of fixtures/helper scripts by consolidating related components into scripts/modules (e.g. `common.platform.transceiver` has all the transceiver plugins).